test/e2e/custom_tool_test.go

```diff
@@ -232,7 +232,10 @@ func TestCMPDiscoverWithFindGlob(t *testing.T) {
 		}).
 		Path("guestbook").
 		When().
-		CreateApp().
+		CreateFromFile(func(app *Application) {
+			// specifically mention the plugin to use (name is based on <plugin name>-<version>
+			app.Spec.Source.Plugin = &ApplicationSourcePlugin{Name: "cmp-find-glob-v1.0"}
+		}).
```

Comment:
Can we test it both ways? The way I'm reading this change, we'd lose the original discovery test.

Reply:
Have added a separate test for plugin name.